README.md

```diff
@@ -1,23 +1,49 @@
 # Burr
 
-## What is Burr?
-
-Burr is a way to express a state machine (i.e. a graph/flowchart) for data/AI projects. You can (and should!) use it for anything where managing state can be hard. Hint: managing state
-is always hard! This is true across a wide array of contexts, from building RAG applications to power a chatbot, to running ML parameter tuning/evaluation workflows, to conducting a complex forecasting simulation.
+Burr makes it easy to develop applications that make decisions based on state (chatbots, agents, simulations, etc...) from simple python building blocks.
+Burr includes a UI that can track/monitor those decisions in real time.
 
 Link to [documentation](https://burr.dagworks.io/). Quick video intro [here](https://www.loom.com/share/8a92474bb7574d6eb4cd25c21913adf2).
 
-Burr is:
+## 🏃Quick start
+
+Install from `pypi`:
+
+```bash
+pip install "burr[start]"
+```
+
+Then run the server:
+
+```bash
+burr
+```
+This will open up a demo -- to chat it requires the `OPENAI_API_KEY` environment variable to be set,
```

Comment:
```suggestion
This will open up the burr tracking UI with demo data and an example built in application -- to use the demo chat application, it requires the `OPENAI_API_KEY` environment variable to be set,
```